Fix consent issue when web scraping

jonnybergdahl opened the pull request:
Add new Consent cookie
Add debug logging - See [this](https://github.com/custom-components/youtube/issues/56) and [this](https://github.com/custom-components/youtube/issues/65)
Code cleanup

bergdahl committed:
Add new Consent cookie
Add debug logging
Code cleanup

bergdahl committed:
Added support for @ChannelName style channel id's.

jonnybergdahl commented:
Added support for using channel names instead of channel id. 

Note that you must use quotes for those though:
```- platform: youtube
  channel_id: '@UnexpectedMaker'```

bergdahl committed:
Removed async_timeout after consulting with @joostlek

jonnybergdahl commented:
I have also removed the use of async_timeout after consulting with @joostlek

bergdahl committed:
Update README for channel name

daFreeMan commented:
What are the odds of this ever getting finished and the PR being accepted? Looks like there are several reported issues that this will resolve. It would be fantastic if it did and made this useful.

jonnybergdahl commented:
I use mine instead - https://github.com/jonnybergdahl/youtube
Has worked flawlessly for almost 2 years now.

bergdahl committed:
Fix channel picture

jonnybergdahl committed:
Remove zip_release and filename from hacs.json

jonnybergdahl committed:
Update YouTube Sensor manifest for version 1.0.2

Updated version to 1.0.2 and changed documentation and codeowner details.

jonnybergdahl committed:
Update README with fork details and improvements

This is a forked version of the original YouTube integration, fixing several issues and adding support for new features.

jonnybergdahl committed:
Update YouTube Sensor version to 1.0.3
